Cargo.lock

```diff
@@ -2,7 +2,10 @@
 name = "rstreediff"
 version = "0.1.0"
 dependencies = [
+ "dot 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
OK, well I don't know what the lock file is for (dep versions are listed in Cargo.toml). I suppose do what the rust devs do.